pkg/ui/src/views/app/components/layoutSidebar/index.tsx

```diff
@@ -10,29 +10,49 @@
 
 import React from "react";
 import { connect } from "react-redux";
-import { Link, withRouter, WithRouterProps } from "react-router";
+import { Link, withRouter, RouteComponentProps } from "react-router-dom";
+
 import { SideNavigation } from "src/components";
 import "./navigation-bar.styl";
 
+interface RouteParam {
+  path: string;
+  text: string;
+  activeFor: string[];
+  // ignoreFor allows exclude specific paths from been recognized as active
+  // even if path is matched with activeFor paths.
+  ignoreFor?: string[];
+}
+
 /**
  * Sidebar represents the static navigation sidebar available on all pages. It
  * displays a number of graphic icons representing available pages; the icon of
  * the page which is currently active will be highlighted.
  */
-class Sidebar extends React.Component<{} & WithRouterProps> {
-  readonly routes = [
+class Sidebar extends React.Component<RouteComponentProps> {
+  readonly routes: RouteParam[] = [
     { path: "/overview", text: "Overview", activeFor: ["/node"] },
     { path: "/metrics", text: "Metrics", activeFor: [] },
     { path: "/databases", text: "Databases", activeFor: ["/database"] },
     { path: "/statements", text: "Statements", activeFor: ["/statement"] },
     { path: "/reports/network", text: "Network Latency", activeFor: ["/reports/network"] },
     { path: "/jobs", text: "Jobs", activeFor: [] },
-    { path: "/debug", text: "Advanced Debug", activeFor: ["/reports", "/data-distribution", "/raft"] },
+    {
+      path: "/debug",
+      text: "Advanced Debug",
+      activeFor: ["/reports", "/data-distribution", "/raft"],
+      ignoreFor: ["/reports/network"],
+    },
   ];
 
   isActiveNavigationItem = (path: string): boolean => {
-    const { activeFor } = this.routes.find(route => route.path === path);
-    return [...activeFor, path].some(p => this.props.router.isActive(p));
+    const { pathname } = this.props.location;
+    const { activeFor, ignoreFor = []} = this.routes.find(route => route.path === path);
+    return [...activeFor, path].some(p => {
+      const isMatchedToIgnoredPaths = ignoreFor.some(ignorePath => pathname.startsWith(ignorePath));
```

Comment:
`ignoreFor` is necessary to manage one tricky condition which we have with `Network Latency` view.
`Network Latency` page has `/reports/network` path (since this view was under `Debug` page for a while).

`Debug` path --> `path: "/debug"` and activeFor: ["**/reports**", ...]
`Network` path --> `path: "/reports/network"` and activeFor: ["**/reports**/network"]

`/reports` path (for Debug page) includes "/reports/network" as well. As result two links (Debug and Network Latency) are set as active. To avoid this, I decided to extend this configuration with `ignoreFor` option.

I totally agree with you ... this PR isn't a good place to introduce these changes.

Reply:
Gotcha. I'll make a separate ticket to clean up the path discrepancy later. Thanks for the detailed explanation!